run_unit_tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+VERSION="dev"
+SHA=$(git rev-parse --verify HEAD)
+BUILDDATE=$(date '+%Y/%m/%d %H:%M:%S %Z')
+GOVERSION=$(go version)
+LDFLAGS="-X 'main.version=${VERSION}' -X 'main.sha=${SHA}' -X 'main.builddate=${BUILDDATE}' -X 'main.goversion=${GOVERSION}'"
+
+go test $(go list ./... | grep -v /vendor/) 
+
+echo "Unit tests: Done"
```

Comment:
We will do this one better: the file has been deleted, as the "go test ..." line is now run as a one-line script in a docker container.